gprofiler/utils/__init__.py

```diff
@@ -457,3 +457,12 @@ def is_pyinstaller() -> bool:
 
 def get_staticx_dir() -> Optional[str]:
     return os.getenv("STATICX_BUNDLE_DIR")
+
+
+def add_permission_dir(path: str, permission_for_file: int, permission_for_dir: int) -> None:
```

Comment:
Instead of this approach - you can use `shutil.copytree(copy_function=...)` and as your copy function, pass a function that calls `copy2` which is the one used by default, and then follows by the needed chmod.

Reply:
I see. Okay, let's keep this function for now. Leaving the other comment I had on it.